### PR TITLE
Add link to compiled binaries in readme

### PR DESCRIPTION
**Pull request by kokarn**
I actually missed this when first trying it out. 
Thought it was really weird that you had to run it from the CLI....

Not really sure about the text tho. 
Any suggestions? 
Link directly to the compiled files?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ James is build with [hoxy](https://github.com/greim/hoxy), [electron](https://gi
 
 ![](resource/screenshot-1.png)
 
+## Installing
+Pre-compiled binaries are available under the [releases](https://github.com/james-proxy/james/releases) tab.
+
+Just download the appropriate one for your system and run it.
```
Comment (mitchhentges) [inline README.md]: For the sake of terseness, not sure that this line is necessary.

**Comment by mitchhentges** [comment]
Awesome, more contributors!
That's actually a pretty good call. Not to put any blame on GitHub, but the "releases" tab is a little out-of-the-way.

**Comment by kokarn** [comment]
I did a complete re-make of it. 
The copy still doesn't feel 100% but I really think we should have some sort of button for it. More options are available at the bottom of [shields.io](http://shields.io/)

File: README.md [hunk 2]
```diff
@@ -9,6 +9,15 @@ James is build with [hoxy](https://github.com/greim/hoxy), [electron](https://gi
...
 ![](resource/screenshot-1.png)
 
+## Installing
+Download the correct version for your OS and run
+
+![](https://img.shields.io/badge/download-OSX-green.svg?link=https://github.com/james-proxy/james/releases/download/v1.3.2/James-darwin-x64.zip)
+![](https://img.shields.io/badge/download-linux--ia32-green.svg?link=https://github.com/james-proxy/james/releases/download/v1.3.2/James-linux-ia32.zip)
+![](https://img.shields.io/badge/download-linux--x64-green.svg?link=https://github.com/james-proxy/james/releases/download/v1.3.2/James-linux-x64.zip)
+![](https://img.shields.io/badge/download-windows--ia32-green.svg?https://github.com/james-proxy/james/releases/download/v1.3.2/James-win32-ia32.zip)
+![](https://img.shields.io/badge/download-windows--x64-green.svg?https://github.com/james-proxy/james/releases/download/v1.3.2/James-win32-x64.zip)
```
Comment (mitchhentges) [inline README.md]: Each shield should link to each file, right now clicking on a shield simply opens the image in a new tab :)

**Comment by kokarn** [comment]
Sorry about that, I was apparently playing a bit fast and loose with the markdown for them ;)

**Comment by mitchhentges** [comment]
:+1: from me!
screenshot

**Comment by kokarn** [comment]
If people are interested there are a bunch of cool things you can do with shields.io, like custom icons, colors, texts whatever. So if anybody has any suggestions, I'll see what I can do.

**Comment by nerdbeere** [comment]
Very cool, thank you

**Comment by nerdbeere** [comment]
It would be even cooler if the links would point to the latest releases so we don't have to change it for every new release.

**Comment by kokarn** [comment]
Sure thing, if you can find any url for the binaries. AFAIK the link that page refers to is just the general releases page, not the individual files. 

Tried 
https://github.com/james-proxy/james/releases/latest/download/James-darwin-x64.zip
https://github.com/james-proxy/james/releases/download/latest/James-darwin-x64.zip
https://github.com/james-proxy/james/releases/latest/James-darwin-x64.zip
but none of them work. 

Googling and SO only shows results that use a script to get the latest release and insert that or linking to a local file in the repository.

**Comment by mitchhentges** [comment]
Hmm. The cost of updating the list on every version-change is pretty brutal.
I'd vote towards having it like you originally made the PR: a link to the Releases page.

**Comment by nerdbeere** [comment]
Ok then I'd also say we should just link to the releases page.

**Comment by kokarn** [comment]
How do you currently create releases? 
Would be pretty trivial to write something that just does some search/replace for old/new version in the readme if you create a new version tho.

**Comment by mitchhentges** [comment]
Well, it's kinda pro/con. One extra click to view the Releases page versus. release-time complexity. I'm still feeling linking to releases

**Comment by kokarn** [comment]
Sure thing, if @nerdbeere agrees i'll update my PR

**Comment by nerdbeere** [comment]
It would be awesome if you could do that.
In general the build and release process should be as automated as possible.
Thanks for your contribution!

**Comment by mitchhentges** [comment]
@kokarn, just a reminder, if this slipped through your notifications: I'd love to merge this in, if you could make the changes :)

**Comment by kokarn** [comment]
@mitchhentges Nah, it's just on my todo. 
I'll try to make the changes today so it's ready for a merge then.

**Comment by mitchhentges** [comment]
Awesome, thanks :)

**Comment by kokarn** [comment]
Heres how it's looking right now

**Comment by mitchhentges** [comment]
:chicken: thanks man :)

**Comment by kokarn** [comment]
:cake:

**Pull request merged**